tests/test_bot.py

```diff
@@ -1263,15 +1263,22 @@ def test_set_chat_description(self, bot, channel_id):
     @flaky(3, 1)
     @pytest.mark.timeout(10)
     def test_pin_and_unpin_message(self, bot, super_group_id):
-        message = bot.send_message(super_group_id, text="test_pin_message")
+        message1 = bot.send_message(super_group_id, text="test_pin_message_1")
+        message2 = bot.send_message(super_group_id, text="test_pin_message_2")
+        message3 = bot.send_message(super_group_id, text="test_pin_message_3")
+
         assert bot.pin_chat_message(
-            chat_id=super_group_id, message_id=message.message_id, disable_notification=True
+            chat_id=super_group_id, message_id=message1.message_id, disable_notification=True
         )
+        message2.pin()
+        message3.pin()
```

Comment:
using `bot.pin_chat_message` without passing `message_id` should also cover the uncovered line marked by codecov

Reply:
uhm, how will the message be pinned without the `message_id`? Isn't it required? Did you mean `unpin_chat_message`?

Reply:
uh, yeah, mixed that up, sry. anyway, that if-clause branch in `unpit_chat_message` should be tested :D